libsoftwarecontainer/src/softwarecontainer.cpp

```diff
@@ -444,7 +439,7 @@ ReturnCode SoftwareContainer::bindMount(const std::string &pathOnHost, const std
         throw InvalidOperationError(message);
     }
 
-    return getContainer()->bindMountInContainer(pathOnHost, pathInContainer, readonly);
+    return isSuccess(getContainer()->bindMountInContainer(pathOnHost, pathInContainer, readonly));
```

Comment:
This is also a function in the container, that hasn't been changed yet. As mentioned above, I felt I needed to make an intermediate commit.